localisation/00_wc_vanilla_replacements.csv

```diff
@@ -340,3 +340,11 @@ EVTOPTCHFP21041;I will tell [From.GetHerHim] of my prosperous realm.;Je vais lui
 EVTOPT_HFP_11011_B;Wait! How about you just take this [equipped_artifact.GetName] and let me live?!;Attendez�! Et si vous preniez ��[equipped_artifact.GetName]�� et me laissiez en vie�?;Nehmt [equipped_artifact.GetName] uns lasst mich leben!;;�Alto! �Y si os llev�is este/a [equipped_artifact.GetName] y me dej�is vivir?;;;;;;;;;x
 less_likely_to_be_looted_modifier_tt;�YArtifacts�! are less likely to be stolen;Les �Yartefacts�! sont moins facilement vol�s;�YArtefakte�! werden weniger wahrscheinlich gestohlen;;Es m�s probable que roben los �Yartefactos�!;;;;;;;;;x
 EVTDESC_HFP_16006_combat_weapon;Upon inspecting the [inherited_artifact.GetArtifactStorage], my guards have found [inherited_artifact.GetName], an old weapon. "It must have belonged to [previous_owner.GetTitledFirstName]", they inform me. "Shame it's not in better shape", they mutter.;Lors de l'inspection de [inherited_artifact.GetArtifactStorage], mes gardes ont trouv� � [inherited_artifact.GetName] �, une arme ancienne. ��Cela devait appartenir [previous_owner.Get_au_TitledFirstName]��, m'informent-ils. ��Dommage que ce ne soit pas en meilleur �tat��, marmonnent-ils.;Bei der Inspektion des [inherited_artifact.GetArtifactStorage] haben meine Wachen [inherited_artifact.GetName], eine alte Waffe, gefunden. �Sie muss [previous_owner.GetTitledFirstName] geh�rt haben�, informieren sie mich. �Schade, dass sie nicht besser erhalten ist�, murmeln sie.;;Al examinar el [inherited_artifact.GetArtifactStorage], mis guardias han encontrado [inherited_artifact.GetName], un arma antigua. Me informa que debi� de pertenecer a [previous_owner.GetTitledFirstName], murmurando que es una pena que no est� en buen estado.;;;;;;;;;x
+REL_LOVER_MALE;lover;amant;Geliebter;;amante;;;;;;;;;x
+REL_LOVER_FEMALE;lover;amante;Geliebte;;amante;;;;;;;;;x
+depopulated_1_desc;This county has been depopulated to a minor extent. As times improve it will repopulate quickly.;La population de ce comt� a chut� suite � une �pid�mie. Quand les maladies ne le frapperont plus, celui-ci se repeuplera rapidement.;Die Bev�lkerungsanzahl in dieser Provinz ist leicht zur�ckgegangen. Wenn sich die Ziten bessern, wird sich die Bev�lkerung schnell erholen.;;Esta provincia ha sido despoblada a escala menor a causea de una epidemia. Cuando la enfermedad deje la provincia, se repoblar� r�pidamente.;;;;;;;;;x
```

Comment:
This one is just my preference, but I think slightly depopulated sounds better than depopulated to a minor extent.
```suggestion
depopulated_1_desc;This county has been slightly depopulated. As times improve it will repopulate quickly.;La population de ce comt� a chut� suite � une �pid�mie. Quand les maladies ne le frapperont plus, celui-ci se repeuplera rapidement.;Die Bev�lkerungsanzahl in dieser Provinz ist leicht zur�ckgegangen. Wenn sich die Ziten bessern, wird sich die Bev�lkerung schnell erholen.;;Esta provincia ha sido despoblada a escala menor a causea de una epidemia. Cuando la enfermedad deje la provincia, se repoblar� r�pidamente.;;;;;;;;;x
```